SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1845,7 +1845,7 @@ void USpatialReceiver::HandleRPC(const Worker_ComponentUpdateOp& Op)
 		const TWeakObjectPtr<UObject> ActorReceivingRPC = PackageMap->GetObjectFromEntityId(Op.entity_id);
 		if (!ActorReceivingRPC.IsValid())
 		{
-			UE_LOG(LogSpatialReceiver, Error,
+			UE_LOG(LogSpatialReceiver, Log,
```

Comment:
Do we still want to log this? I think native wouldn't say anything, right?

Reply:
Yeah my intuition is to leave this log in since the case with actor deleted with RPCs in flight might not happen that often but if this ever helps debug a future issue with crossing boundaries or something it might be worth it.